fitnesse/resources/bootstrap-plus/js/bootstrap-plus.js

```diff
@@ -89,6 +92,9 @@ function processSymbolData(str) {
     DOCUMENT READY START
 */
 $( document ).ready(function() {
+    if ((location.pathname === '/FrontPage' || location.pathname === '/' ) && !location.search.includes('?')) {
+        getVersionData(versionCheck,Window.location + "/?mavenVersions");
```

Comment:
waarom gebruik je hier location en Window.location door elkaar? Ik denk dat `Window.` niet nodig is in deze (je draait al in het window)

Reply:
ik zie dat window location en location eigenlijk dezelfde output geven, zal het even aanpassen